apps/epic-web/src/inngest/inngest-config.ts

```diff
@@ -0,0 +1,23 @@
+import {inngest} from 'inngest/inngest.server'
+import {processNewTip} from 'inngest/functions/tips/process-new-tip'
+import {addSrtToMuxAsset} from 'inngest/functions/tips/add-srt-to-mux-asset'
+import {stripeCheckoutCompleted} from 'inngest/functions/stripe/checkout-completed'
+import {lessonCompleted} from 'inngest/functions/progress/lesson-completed'
+import {writeAnEmail} from 'inngest/functions/ai-email/write-an-email'
+import {purchaseTransferComplete} from 'inngest/functions/purchase/purchase-transfer-complete'
+import {productFunctions} from 'inngest/functions/sanity/product'
+
+export const inngestConfig = {
```

Comment:
this was stuffed in `/apps/api/inngest/route.ts` so I lifted it to here so ALL inngest edits happen in `/inngest`